mne/tests/test_event.py

```diff
@@ -405,4 +412,31 @@ def test_define_events():
     assert_array_equal(true_lag_fill, lag_fill)
     assert_array_equal(true_lag_nofill, lag_nofill)
 
+
+@testing.requires_testing_data
+def test_elekta_averager():
+    """Test averaging according to Elekta DACQ parameters"""
+    raw = io.read_raw_fif(fname_raw_elekta, preload=True)
+    eav = ElektaAverager(raw.info)
+    for cat in eav.categories:
+        # XXX datasets match only when baseline is applied to both,
+        # not sure where relative dc shift comes from
```

Comment:
It's an actual dc shift, I checked. May be caused by Elekta's own averager code, nobody seems to know for certain.